api/types/types.proto

```diff
@@ -620,6 +618,49 @@ message ClusterNetworkingConfigSpecV2 {
         [ (gogoproto.jsontag) = "session_control_timeout", (gogoproto.casttype) = "Duration" ];
 }
 
+// SessionRecordingConfigV2 contains session recording configuration.
+message SessionRecordingConfigV2 {
+    option (gogoproto.goproto_stringer) = false;
+    option (gogoproto.stringer) = false;
+
```

Comment:
If you allow gogoproto to generate a `String` method, you shouldn't need to define it yourself.
```suggestion
```
It looks like these stringer=false options were originally used for backwards compatibility, but they shouldn't be used unless needed.